### PR TITLE
removed optional tabs on the edit post form

### LINKED ISSUE #2127: Posts should NOT be allowed to be  edited to 'Requesting Help' ,if its originally created for 'Offering Help'
Posts should NOT be allowed to be edited to 'Requesting Help' ,if its originally created for 'Offering Help' and Vice Versa.
. 
Steps: 1.Create a post using 'Create a New Post' from Help Board.

2. Choose 'Offering Help'. ENter details and Save
3. Now choose Edit and change to "Requesting Help" tab      and save

Expected:Posts should NOT be allowed to be edited to 'Requesting Help' ,if its originally created for 'Offering Help' and Vice Versa.
.

### PR DESCRIPTION
**Pull request by kamarenmccord**
### What does this pull request aim to achieve? 💯
Removed Tabs from the edit Post form. Now only shows both tabs on the create post and on the edit post form only the chosen (request/offer) is shown.

_Please be concise and link any related issue(s):_
#2127

### CODE REVIEW

**Review by galeza — DISMISSED** [review]
LGTM
I tested on desktop and Android both Chrome browser.
Screenshot_2021-01-15-14-42-09-046_com android chrome

Tested for Offer and Request Help. Create and Edit posts

File: client/src/components/CreatePost/Form/TabForms.js [hunk 1]
```diff
@@ -27,18 +27,31 @@ const ModalComponent = ({
...
+      key:"request",
       title: t("post.gettingHelp"),
       question: t("post.whatNeed"),
-    },
-  ];
+    }
+  }
+
+  if (currentPost){
+    if (currentPost.objective){
```
Comment (ddveloper) [inline client/src/components/CreatePost/Form/TabForms.js]: just for corner case when `currentPost` is not null but `currentPost.objective` is null, maybe combine them to be like:
```
if (currentPost?.objective)
```

Comment (kamarenmccord) [reply client/src/components/CreatePost/Form/TabForms.js]: That's an <strong>excellent</strong> idea. Thanks for the tip! I pushed the branch.

**Review by ddveloper — APPROVED** [review]
LGTM

**Comment by rabbi08** [comment]
Thanks @kamarenmccord 
Verified the fix in the feature branch and is working fine. Please see below GIF:

ezgif com-video-to-gif-2

**Pull request merged**